File: ILT.IHR.DTO/ProcessDataTicket.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ILT.IHR.DTO
{
   public class ProcessDataTicket : AbstractDataObject
    {
        public int ProcessDataTicketID
        {
            get { return base.RecordID; }
            set { base.RecordID = value; }
        }
        public int ProcessDataID { get; set; }
        public int TicketID { get; set; }
        public string Title { get; set; }
        public string TicketDescription { get; set; }
        public int StatusID { get; set; }
        public string Status { get; set; }
        public DateTime? ResolvedDate { get; set; }
        public int AssignedToId { get; set; }
        public string AssignedToUser { get; set; }
    }
}

File: ILT.IHR.APP/ClientApp/src/app/core/interfaces/ProcessDataTicket.ts
export class IProcessDataTicket {
    ProcessDataTicketID: number;
    ProcessDataID: number;
    TicketID: number;
    Title?: string;
    TicketDescription?: string;
    StatusId: number;
    Status?: string;
    ResolvedDate?: Date;
    AssignedToId: number;
    AssignedToUser: string;
    CreatedBy: string;
    CreatedDate: Date;
    ModifiedBy: string;
    ModifiedDate: Date;
    TimeStamp: ['']

}
export class IProcessDataTicketDisplay {
    processDataTicketID: number;
    processDataID: number;
    ticketID: number;
    title?: string;
    ticketDescription?: string;
    statusId: number;
    status?: string;
    resolvedDate?: Date;
    assignedToId: number;
    assignedToUser: string;
    createdBy: string;
    createdDate: Date;
    modifiedBy: string;
    modifiedDate: Date;
    timeStamp: ['']
}
